gettime.c

```diff
@@ -564,7 +564,6 @@ struct clock_thread {
 	int cpu;
 	int debug;
 	pthread_mutex_t lock;
-	pthread_mutex_t started;
```

Comment:
I think we should change this to be a fio_mutex instead. The mutex is used to synchronize startup of the clock threads.

Reply:
What should the behavior be of the actions on the 'started' synchronization object? The unlock from the clock threads on t->started succeeds immediately. I don't see how the actions on 'started' impose any ordering constraints.

Reply:
By the time we end up looping around and unlocking t->lock to kick off the threads at the same time, we know they are all waiting there. It makes us independent of thread startup time.